### ISSUE TITLE
hello

### ISSUE THREAD

**Issue by terryjyu** [raghulkrishna/lending-loan-data-club-dashboard#1]
Hi,

Your project looks really interesting! However, it looks like I can't find the source code. Could you please upload the source code for the plotly dash? 

 Thanks!
Terry

**Comment by raghulkrishna** [comment]
hey terry i will upload the codes in a week than you showing interest in my project
thank you 
krish

**Comment by terryjyu** [comment]
Thanks for the reply! Any chance that you will post it this week? I'm a data science student and learning Plotly with dashboard and find it very helpful material! Actually I find it quite hard to set up a Plotly.io built-in multi- aggregation dropdown using my own dataset since it requires the data to be dictionary type. And I can't find anywhere else that can provide some examples.
Hope you will post earlier!

Thanks!
Terry

**Comment by raghulkrishna** [comment]
Yeah sure give me few days I have some projects going on sorry for that
